network/datadog_checks/network/network.py

```diff
@@ -309,6 +309,15 @@ def _check_linux(self, instance):
                 # Try using `ss` for increased performance over `netstat`
                 ss_env = {"PROC_ROOT": net_proc_base_location}
 
+                # By providing the environment variables in ss_env, the PATH will be overriden. In CentOS,
+                # datadog-agent PATH is "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin", while sh PATH
+                # will be '/usr/local/bin:/usr/bin'. In CentOS, ss is located in /sbin and /usr/sbin, not
+                # in the sh PATH, which will result in network metric collection failure.
+                #
+                # The line below will set sh PATH as the datadog-agent PATH to fix that issue.
+                if os.environ["PATH"] is not None:
```

Comment:
```suggestion
                if "PATH" not in os.environ:
```
The previous code would crash if PATH is not in the map.

Reply:
Thanks! A new change is being committed.

Also added a debug output for out / err in get_subprocess_output